sshd_config.jinja

```diff
@@ -23,6 +23,10 @@ PermitRootLogin no
 ClientAliveInterval {{ client_alive_interval }}
 {%- endif %}
 
+{%- if max_sessions %}
+MaxSessions {{ max_startups }}
```

Comment:
Is it intentional to use `max_startups` here?

Reply:
fixed

Reply:
You need to add it to init.sls, too

Reply:
added